shared/src/proxies/trialSessions/getBulkSpecialTrialSessionCopyNotesProxy.ts

```diff
@@ -0,0 +1,20 @@
+import { post } from '../requests';
+
+/**
```

Comment:
We do not use JSDocs anymore (older pattern) since we moved to Typescript. Feel free to delete the JSDOCS and add types for the inputs and outputs of this function.